test/e2e/conf.js

```diff
@@ -63,15 +63,8 @@ if (process.env.TRAVIS  || process.env.CHROME_HEADLESS) {
   if (process.env.SAUCE_PLATFORM_NAME === 'windows') {
     config.multiCapabilities =
     [{
-      'browserName': 'internet explorer',
```

Comment:
why delete IE?

Reply:
I was testing this out to ensure edge 17 is being picked up by saucelabs.
For some weird reason, it's still picking edge 88. 
I will add ie back after that issue is fixed.